code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -91,6 +91,13 @@
 			visible_message("<span class='notice'>[src] nibbles through the [cheese], attracting another mouse!</span>")
 			qdel(cheese)
 			return
+	for(var/obj/item/reagent_containers/food/snacks/royalcheese/bigcheese in range(1, src))
```

Comment:
what happens if there's a pile of, like, 5 royal cheeses on a tile and a single mouse approaches it?